.devcontainer/devcontainer.json

```diff
@@ -16,7 +16,8 @@
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
 		"ms-vscode.cpptools",
-		"ms-vscode.cmake-tools"
+		"ms-vscode.cmake-tools",
+		"GitHub.copilot"
```

Comment:
Let's drop the copilot, not everyone will have this.

Reply:
This has been removed.